content/articles/dnssec.markdown

```diff
@@ -72,21 +72,29 @@ Click on the Delete DNSSEC Configuration button to remove the zone signing and t
 
-## Key Rotation
+## Key rotation
 
-DNSimple currently rotates both key signing keys and zone signing keys every 90 days. Auto-rotation is mandatory, and you cannot disable it.
+DNSimple rotates both key signing keys and zone signing keys every 90 days. Auto-rotation is mandatory, and you can't disable it.
 
-- If your domain is both registered with us and uses our authoritative name servers: we handle rotation of keys automatically.
-- If your domain is NOT registered with us or DOES NOT uses our authoritative name servers: you'll receive an email notification whenever key rotation starts, with the new DS record, **and you'll need to rotate the DS records at your domain registrar within 3 days**.
+- If your domain is registered with us and uses our authoritative name servers: we handle rotation of keys automatically.
+- If your domain is NOT registered with us or DOES NOT uses our authoritative name servers: you'll receive an email notification with the new DS record whenever key rotation starts, **and you'll need to rotate the DS records (remove the old record and add the new record) at your domain registrar within 3 days**.
 
 <warning>
 #### Failure to update the DS record at your registrar will result in downtime
 
-Please consider carefully whether you are able and willing to rotate DS records at your registrar if your domain is not registered with DNSimple. It is essential that DS records are updated whenever DNSSEC keys are rotated in your DNSimple zone. If you do not update your DS record when your keys change, then your domain will fail to resolve through resolvers that verify DNSSEC keys, including Google's Public DNS. This will result in failed DNS resolution for your domains.
+Please carefully consider whether you're able and willing to rotate DS records at your registrar if your domain is not registered with DNSimple. DS records MUST be updated whenever DNSSEC keys are rotated in your DNSimple zone. If you don't update your DS record when your keys change, your domain will fail to resolve through resolvers that verify DNSSEC keys, including Google's Public DNS. This will result in failed DNS resolution for your domains.
 </warning>
 
 During the key rotation, both the old and new keys are attached to your zone for 3 days (the duration of the key rotation period). At the end of the rotation period our system removes the old key, leaving only the new key in place.
 
+### Automating key rotation
+
+If your domain registrar provides an API for managing DS records, you can automate rotation for domains registered outside of DNSimple. To do this you must use the `dnssec.rotation_start` and `dnssec.rotation_complete` webhook events. More information is available in [our developer documentation](https://developer.dnsimple.com/v2/webhooks/).
+
+### Automating key rotation with other domain registrars
+
+Our system will deliver a webhook call when DNSSEC rotation is started and completed. You may 
```

Comment:
>Our system will deliver a webhook call when DNSSEC rotation is started and completed. You may 

You may... ?

Reply:
Extra stuff made it through. I've removed it.